app/models/todo_item.rb

```diff
@@ -1,7 +1,35 @@
 class TodoItem < ApplicationRecord
   belongs_to :todo_list
 
+  after_create :assign_status
+
+  enum status: {red: 0, yellow: 1, blue: 2, lightgreen: 3}
```

Comment:
Status names are not descriptive of their priority,
Could've used, Unset, Low, Medium & High
This provides increased readability

Reply:
Yeah, That is some sort of hack as this color is actually I am using as color for the priority as well. So To make development fast, I just used that. That's the only thing.